### ISSUE TITLE
Testing with webpack loaders

### ISSUE THREAD

**Issue by primozs** [istarkov/babel-plugin-webpack-loaders#30]
This module is great for testing when you need webpack loader while running tests...
Thx
Cheers

**Comment by istarkov** [comment]
When I wrote this plugin, I didn't thought that this plugin will be used for testing. 
I wrote it to create standalone builds of our internal libraries ;-)

**Comment by karlhorky** [comment]
Yeah, we could document this a bit - add a section in the readme referencing `mocha` or something and an example.

**Comment by iamstarkov** [comment]
i do appreciate idea of creating guide about "how to setup this plugin to use with mocha for testing react components"

**Comment by istarkov** [comment]
http://browniefed.com/blog/2016/01/31/css-modules-webpack-and-testing/
